ads/google/a4a/utils.js

```diff
@@ -115,9 +116,14 @@ export function googleAdUrl(
     const viewportRect = viewport.getRect();
     const iframeDepth = iframeNestingDepth(win);
     const viewportSize = viewport.getSize();
-    if (ValidAdContainerTypes.indexOf(adElement.parentElement.tagName) >= 0) {
-      queryParams.push({name: 'amp_ct',
-                        value: adElement.parentElement.tagName});
+    let parentElement = adElement.parentElement;
+    const containerTypes = [];
```

Comment:
Using an array would allow a container type to appear multiple times but I believe you want a set, right?  Consider using an object (value does not matter)